word2vec/data/getdata.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
Also add `.gitignore` for getdata.sh generated files.

![image](https://cloud.githubusercontent.com/assets/728699/21561086/ce599eda-cea2-11e6-8e8d-8f4f332bb971.png)